### DISCUSSION TITLE
openTCS now on GitHub!

### DISCUSSION THREAD

**Discussion by swltr** [openTCS/opentcs#1] [Announcements]
Hello everyone,

in the past, we have occasionally been asked about having an openTCS project on GitHub, with a public source code repository. As coders, we have always looked forward to that step, but for various reasons, we continuously needed to defer that move. Now, however, it has finally happened, and the openTCS team at Fraunhofer IML is glad to be able to announce that openTCS is now on GitHub!

We hope you like it here, and we're looking forward to see you around!

Best regards,
the openTCS development team at Fraunhofer IML

**Comment by sweetquiet** [comment]
wow.cool！

**Comment by DarrenGB** [comment]
This looks awesome 
Where to start, with any open source technology the following information is required.
When you say open source, does this meaning you already have a GUI, PCB and a list of base components to get started ?
Are we talking about SLAM, GNSS, laser, deep learning etc 
Do I have access to the GUI and hardware requirements that I may build my own platform 
Kind regards 
Darren 
Melbourne Australia

**Reply by swltr** [reply]
@DarrenGB On the openTCS homepage (https://www.opentcs.org/), which is linked to in the "Getting Started" section of this project's readme file, for example, you will find the project's documentation. It contains a User's Guide with a general introduction of the software, explaining what it is intended to do, and also a Developer's Guide and API documentation.

(Please note that it is better to start a new thread in the discussion forum for questions.)
